integration/travis/scheduler_config.edn

```diff
@@ -16,6 +16,10 @@
          :failover-timeout-ms nil
          :leader-path "/cook-scheduler"
          :role "cook"}
+ :executor {:command "/home/travis/build/twosigma/Cook/travis/cook-executor"
```

Comment:
Yes, but this mimics a use case where we want to use a command available on the agent.

The simulator uses an example of downloading it (I couldn't get http uri working, so I manually curl/wget it there).